### ISSUE TITLE
When should we instantiate epo:ProcedureSpecificTerm?

### ISSUE THREAD

**Issue by csnyulas** [OP-TED/ted-rdf-mapping#332]
In the `tedm:ProcedureSpecificTerm` TriplesMap we instantiate the class `epo:ProcedureSpecificTerm` for all Procedure instances (without any additional condition other than the presence of the `CONTRACTING_BODY` XPATH), and we attach some information to it in `s2_object.rml.ttl` such as `epo:hasCrossBorderLaw` or `epo:isAwardedByCPB`, if corresponding information is present in the input data. This instance of the `epo:ProcedureSpecificTerm` will be linked to the instance of the `epo:Procedure` through the property `epo:isSubjectToProcedureSpecificTerm`
![image](https://user-images.githubusercontent.com/2441046/214980272-ec27cd6a-3f8a-4295-8b47-cfe6728ba7fa.png)

We should check:
- if it is ever necessary, and if yes when, to instantiate the class  `epo:ProcedureSpecificTerm` itself, or should we rather instantiate only its subclasses
- to what, and through which property, to link such `epo:ProcedureSpecificTerm` instance
- how can we eliminate creation of "empty" `epo:ProcedureSpecificTerm` instances, which we assume is highly desirable

See for example a screenshot from the generated output for notice **063415-2021** :

![image](https://user-images.githubusercontent.com/2441046/214981296-c3ce28c2-e774-43ef-a9e8-3efc9d9d8382.png)

Closely related to #331 and the first bullet point of #122.

**Comment by csnyulas** [comment]
The answers to these questions can be found in this comment: https://github.com/OP-TED/ted-rdf-mapping/issues/331#issuecomment-1429257196

**TLDR:** We should never instantiate the class `epo:ProcedureSpecificTerm` itself, but only its subclasses, and the instances of those subclasses should be connected to the `epo:Procedure` instance through the through the `epo:isSubjectToProcedureSpecificTerm` property.

**Comment by csnyulas** [comment]
Both conceptual mapping (v.6.6.3) and the technical mapping was updated to fix this issue (see referenced commits).
